src/frontend/src/icp-eth/components/tokens/IcManageTokens.svelte

```diff
@@ -69,40 +70,47 @@
 			);
 
 	let tokens: IcrcCustomToken[] = [];
-	$: tokens = filteredTokens.map(({ ledgerCanisterId, enabled, ...rest }) => ({
-		ledgerCanisterId,
-		enabled: modifiedTokens[`${ledgerCanisterId}`]?.enabled ?? enabled,
+	$: tokens = filteredTokens.map(({ id, network, enabled, ...rest }) => ({
+		id,
```

Comment:
I think we do not need to explicitely add the `id` here given it is contained in the spread `...rest`

Reply:
i extract it because I use it for the enabled line after:

```svelte
	$: tokens = filteredTokens.map(({ id, network, enabled, ...rest }) => ({
		id,
		network,
		enabled: modifiedTokens[network.id]?.[id]?.enabled ?? enabled,   <--- here
		...rest
	}));
```

Reply:
Gotcha